script.js

```diff
@@ -0,0 +1,164 @@
+//1
+const createTeam = (teamName, driverName, birthYear, car, sponsor, isAvailableDriver) => {
+    const team = {
+        teamName,
+        driverName,
+        birthYear,
+        car,
+        sponsor,
+        isAvailableDriver
+    };
+    return team
```

Comment:
It can be a little shorter
```suggestion
    return {
        teamName,
        driverName,
        birthYear,
        car,
        sponsor,
        isAvailableDriver
    };
```